src/Parser/BitcoinMoneyParser.php

```diff
@@ -41,6 +41,10 @@ public function parse($money, $forceCurrency = null)
             throw new ParserException('Value cannot be parsed as Bitcoin');
         }
 
+        if ($forceCurrency !== null && !$forceCurrency instanceof Currency) {
+            @trigger_error('Passing a currency as string is deprecated since 3.1 and will be removed in 4.0. Please pass a '.Currency::class.' instance instead.', E_USER_DEPRECATED);
+        }
+
```

Comment:
OK I have made the change to BitcoinMoneyParser as well.. just to be sure here,,, `$forceCurrency` is never used inside this parser... so not sure if it makes sense to have a deprecation in here...

Reply:
see #540 , #539